Python 3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 22:39:24) [MSC v.1916 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
====================== RESTART: H:/python/random module.py ======================
Traceback (most recent call last):
  File "H:/python/random module.py", line 2, in <module>
    number = random.randomint(1,1000)
AttributeError: module 'random' has no attribute 'randomint'
>>> 
====================== RESTART: H:/python/random module.py ======================
273
>>> 
================================= RESTART: Shell ================================
>>> 
====================== RESTART: H:/python/random module.py ======================
0.598334614930674
>>> 
====================== RESTART: H:/python/random module.py ======================
['blue', 'red', 'black', 'purple', 'green']
>>> 
====================== RESTART: H:/python/random module.py ======================
['blue', 'red', 'black', 'purple', 'green']
>>> 

====================== RESTART: H:/python/random module.py ======================
>>> 
['blue', 'red', 'black', 'purple', 'green']
>>> 
====================== RESTART: H:/python/random module.py ======================
['blue', 'red', 'black', 'purple', 'green']
>>> 
====================== RESTART: H:/python/random module.py ======================
blue
>>> 
>>> 

====================== RESTART: H:/python/random module.py ======================
>>> 
purple
>>> 

====================== RESTART: H:/python/random module.py ======================
>>> 
purple
>>> 
>>> 

====================== RESTART: H:/python/random module.py ======================
>>> 
red
>>> 
===================== RESTART: H:/python/random module 2.py =====================
guess the outcomeheads
bad luck
['heads', 'tails']
>>> 
===================== RESTART: H:/python/random module 2.py =====================
Traceback (most recent call last):
  File "H:/python/random module 2.py", line 3, in <module>
    outcome = random.choice(outcome)
NameError: name 'outcome' is not defined
>>> 
===================== RESTART: H:/python/random module 2.py =====================
guess the outcomeheads
bad luck
['heads', 'tails']
>>> 
===================== RESTART: H:/python/random module 2.py =====================
guess the outcomeheads
bad luck
heads
>>> 
===================== RESTART: H:/python/random module 2.py =====================
guess the outcomeheads
bad luck
tails
>>> 

===================== RESTART: H:/python/random module 2.py =====================
>>> 
guess the outcomeheads
you win
>>> 

===================== RESTART: H:/python/random module 2.py =====================
>>> 
guess the outcometails
you win
>>> 
>>> 

#===================== RESTART: H:/python/random module 2.py =====================
>>> 
guess the outcomeheads
bad luck
tails
>>> 

===================== RESTART: H:/python/random module 2.py =====================
>>> 
guess the outcomeheads
bad luck
tails
>>> 

===================== RESTART: H:/python/random module 2.py =====================
>>> 
guess the outcomeheads
you win
>>> 
===================== RESTART: H:/python/random module 3.py =====================
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 2, in <module>
    num = random.ranint(1,5)
AttributeError: module 'random' has no attribute 'ranint'
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number3
bad luck
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 8, in <module>
    print(outcome)
NameError: name 'outcome' is not defined
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number3
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 6, in <module>
    elif guess >= num:
TypeError: '>=' not supported between instances of 'str' and 'int'
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number3
Traceback (most recent call last):#
  File "H:/python/random module 3.py", line 6, in <module>
    elif guess > num:
TypeError: '>' not supported between instances of 'str' and 'int'
>>> 

===================== RESTART: H:/python/random module 3.py =====================
>>> 
pick a number2
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 6, in <module>
    elif guess > num:
TypeError: '>' not supported between instances of 'str' and 'int'
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number5
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 6, in <module>
    elif guess > num:
TypeError: '>' not supported between instances of 'str' and 'int'
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number3
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 6, in <module>
    elif guess > num:
TypeError: '>' not supported between instances of 'str' and 'int'
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number3
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 6, in <module>
    elif guess>num:
TypeError: '>' not supported between instances of 'str' and 'int'
>>> 
===================== RESTART: H:/python/random module 3.py =====================
pick a number2
Traceback (most recent call last):
  File "H:/python/random module 3.py", line 8, in <module>
    elif guesse < num:
NameError: name 'guesse' is not defined
>>> 